README.md

```diff
@@ -8,16 +8,59 @@
 GeekoDoc is a RELAX NG schema and a subset of DocBook 5. Currently, it can be
 used in two variants:
 
-1. As the file `geekodoc5.rn{c,g}` which is based on `docbookxi.rn{c,g}`. In
+1. As the file `geekodoc-v?.rn{c,g}` which is based on `docbookxi.rn{c,g}`. In
    other words, the GeekoDoc schema cannot life without the DocBook schema.
-2. As a single`geekodoc5-flat.rn{c,g}`. This file is independant from the
+2. As a single`geekodoc-v?-flat.rn{c,g}`. This file is independant from the
    DocBook schema and can be used without having DocBook 5 installed on
    your system.
 
 Both variants contain the same structure, elements, and attributes. They
 serve different purposes.
 
 
+## Available GeekoDoc Versions
+
+Currently, to support legacy documentation, we split the GeekoDoc schema into
+two versions:
+
+* **Version 1** this is the legacy version.
+* **Version 2** this is where all the new features are added.
+
+The two versions can be accessed by its URI.
+
+* Version 1 URIs
```

Comment:
> This is _really_ confusing because there are now v1 and v2 URIs and v1 and v2 Geekodoc.

Well, if we have two GeekoDoc versions we need also to distinguish them, right?

> In addition, I always used this as the URI value in DAPS: DOCBOOK5_RNG_URI="https://github.com/openSUSE/geekodoc/raw/master/geekodoc/rng/geekodoc5-flat.rnc". This style is not documented at all here ...

Yeah, added a separate section about _Using GeekoDoc with DAPS_.

You can add any URI into the env variable as long as it can be resolved though our (GeekoDoc) XML catalog.